pkg/apis/addons/v1alpha1/common.go

```diff
@@ -51,7 +51,7 @@ type SpecRepository struct {
 type CommonAddonsConfigurationSpec struct {
 	// ReprocessRequest is strictly increasing, non-negative integer counter
 	// that can be incremented by a user to manually trigger the reprocessing action of given CR.
-	ReprocessRequest uint64           `json:"reprocessRequest,omitempty"`
```

Comment:
add info that it should be validated in webhook, and uint cannot be used (+ link to a proper section in convention doc)